### ISSUE TITLE
Add pre-release builds to DockerHub especially the MariaDB version

### ISSUE THREAD

**Issue by paulpoco** [signum-network/signum-node#736]
Add pre-release builds to DockerHub especially the MariaDB version.

This way the Pre-release beta versions for Testnet can be run in a Docker Container.

**Comment by ohager** [comment]
will be done finally

**Comment by damccull** [comment]
I'm fairly confident that the only difference between the h2 and maria images is the default database config file that it copies to the host automatically. I'll have to double check, though.

With this in mind, it may be worth considering dropping the database specific images for db-agnostic main, test, and pre-release variants. This will require some configuration option prior to launch to be manually set, though. I think I did it the current way to make it a simple default.

**Comment by ohager** [comment]
exactly...as you can see in #774 I'm using build args now. The Docker Images will be provided soon. And then the action for pre-release will be added also, so somehting like `signumnode:h2-testnet-3.9.0-prerelease` will be available
